ethereumj-core/src/test/java/org/ethereum/config/SystemPropertiesTest.java

```diff
@@ -518,6 +518,30 @@ public void testDump() {
         assertTrue(StringUtils.countOccurrencesOf(dump, "{") > 50);
     }
 
+    @Test
+    public void testMergeConfigs1() {
+        String firstConfig = "";
+        SystemProperties props = new SystemProperties();
+        Config config = props.mergeConfigs(firstConfig, ConfigFactory::parseString);
+        assertFalse(config.hasPath("peer.listen.port"));
+    }
+
+    @Test
+    public void testMergeConfigs2() {
+        String firstConfig = "peer.listen.port=30123";
+        SystemProperties props = new SystemProperties();
+        Config config = props.mergeConfigs(firstConfig, ConfigFactory::parseString);
+        assertEquals(30123, config.getInt("peer.listen.port"));
+    }
+
+    @Test
+    public void testMergeConfigs3() {
```

Comment:
It lacks of test case with different properties that are eventually merged into single configuration.